neverwinter/nwscript/native/scriptcompparsetree.cpp

```diff
@@ -396,6 +396,14 @@ BOOL CScriptCompiler::CheckForBadLValue(CScriptParseTreeNode *pNode)
 //                as mud.
 ///////////////////////////////////////////////////////////////////////////////
 
+#define PARSER_ERROR(err)                                                     \
+    do {                                                                      \
+        m_sParserErrorExtraInfo.Format(" [%s:%d:%d:%s]",                      \
+            GrammarToString(nTopStackState), nTopStackRule, nTopStackTerm,    \
```

Comment:
Rule and term will be confusing for new readers. Maybe add rule=%d:term=%d in the output? Not that they'd know what it is, but at least it'd be a thread to pull on. 🤷